argocd/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine as build
+# renovate: datasource=github-releases depName=argoproj/argo-cd
+ENV VERSION=2.10.2
```

Comment:
It's easy to forget to keep this version in sync [with the helm chart version](https://github.com/kairos-io/community-bundles/pull/71/files#diff-9d99293b47478b3521f9c2104495f35d9612bfaf09638496e91496f529eb86a1R19) but we can improve it in the future.